spec/v2/providers/alerts/crashlytics.spec.ts

```diff
@@ -8,437 +8,121 @@ const APPID = '123456789';
-          },
-          retry: false,
-        },
-      });
-    });
-
-    it('should create a function with base opts', () => {
-      const func = crashlytics.onVelocityAlertPublished(
-        { ...FULL_OPTIONS },
-        myHandler
-      );
-
-      expect(func.__endpoint).to.deep.equal({
-        ...FULL_ENDPOINT,
-        eventTrigger: {
-          eventType: alerts.eventType,
-          eventFilters: {
-            alertType: crashlytics.velocityAlert,
-          },
-          retry: false,
-        },
-      });
-    });
-
-    it('should create a function with opts', () => {
-      const func = crashlytics.onVelocityAlertPublished(
-        { ...FULL_OPTIONS, appId: APPID },
-        myHandler
-      );
-
-      expect(func.__endpoint).to.deep.equal({
-        ...FULL_ENDPOINT,
-        eventTrigger: {
-          eventType: alerts.eventType,
-          eventFilters: {
-            alertType: crashlytics.velocityAlert,
-            appId: APPID,
-          },
-          retry: false,
-        },
-      });
-    });
-  });
-
-  describe('onNewAnrIssuePublished', () => {
-    it('should create a function only handler', () => {
-      const func = crashlytics.onNewAnrIssuePublished(myHandler);
-
-      expect(func.__endpoint).to.deep.equal({
-        platform: 'gcfv2',
-        labels: {},
-        eventTrigger: {
-          eventType: alerts.eventType,
-          eventFilters: {
-            alertType: crashlytics.newAnrIssueAlert,
-          },
-          retry: false,
-        },
-      });
-    });
-
-    it('should create a function with appId', () => {
-      const func = crashlytics.onNewAnrIssuePublished(APPID, myHandler);
-
-      expect(func.__endpoint).to.deep.equal({
-        platform: 'gcfv2',
-        labels: {},
-        eventTrigger: {
-          eventType: alerts.eventType,
-          eventFilters: {
-            alertType: crashlytics.newAnrIssueAlert,
-            appId: APPID,
-          },
-          retry: false,
-        },
-      });
-    });
-
-    it('should create a function with base opts', () => {
-      const func = crashlytics.onNewAnrIssuePublished(
-        { ...FULL_OPTIONS },
-        myHandler
-      );
-
-      expect(func.__endpoint).to.deep.equal({
-        ...FULL_ENDPOINT,
-        eventTrigger: {
-          eventType: alerts.eventType,
-          eventFilters: {
-            alertType: crashlytics.newAnrIssueAlert,
-          },
-          retry: false,
-        },
-      });
-    });
-
-    it('should create a function with opts', () => {
-      const func = crashlytics.onNewAnrIssuePublished(
-        { ...FULL_OPTIONS, appId: APPID },
-        myHandler
-      );
-
-      expect(func.__endpoint).to.deep.equal({
-        ...FULL_ENDPOINT,
-        eventTrigger: {
-          eventType: alerts.eventType,
-          eventFilters: {
-            alertType: crashlytics.newAnrIssueAlert,
-            appId: APPID,
-          },
-          retry: false,
-        },
-      });
-    });
-  });
+  }
```

Comment:
I love the matrix driven tests. Good job deleting so much code!

![Simpsons flamethrower meme](https://c.tenor.com/0dwA9_hd4d4AAAAC/simpsons-villain.gif)